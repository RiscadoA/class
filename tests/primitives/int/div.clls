proc main() {
    println(0 / 1);
    println(0 / 4232);
    println(0 / -2131);
    println(1 / 230);
    println(1 / 1);
    println(2 / 1);
    println(142 / 1);
    println(142 / 2);
    println(142 / 33);
    println(5 / -2);
    println(-5 / 2);
    ()
};;
