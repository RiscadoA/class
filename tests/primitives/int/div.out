0
0
0
0
1
2
142
71
4
-2
-2
